deployments/Dockerfiles/open_aea/start.sh

```diff
@@ -12,7 +12,8 @@ then
 fi
 
 echo "Loading $VALORY_APPLICATION"
-aea fetch $VALORY_APPLICATION --local --alias agent
+aea init --reset --author valory --ipfs --remote
+aea fetch $VALORY_APPLICATION --remote --alias agent
```

Comment:
I am 100% certain we should be fetching the packages from the remote.

Reply:
Hm then this doesn't work during local deployment - so no.

Reply:
Ok agree, it needs to point to remote!